docker-compose.yml

```diff
@@ -2,25 +2,33 @@ version: "3.9"
 services:
   db:
     image: postgres:10.9
+    environment:
+      POSTGRES_PASSWORD: password
+      POSTGRES_USER: postgres
     volumes:
-      - ./tmp/db:/var/lib/postgresql/data
+      - db_data:/var/lib/postgresql/data
```

Comment:
the database data shouldn't be in the tmp folder, it should be safe to delete any content inside ./tmp